atd-vze/src/views/Crashes/Crash.js

```diff
@@ -44,7 +44,15 @@ import {
 
 function Crash(props) {
   const crashId = props.match.params.id;
-  const { loading, error, data, refetch } = useQuery(GET_CRASH, {
+  const { loading, error, data, refetch } = useQuery(GET_CRASH_OLD, {
+    variables: { crashId },
+  });
+  const {
+    loading: crashLoading,
+    error: crashError,
+    data: crashData,
+    refetch: crashRefetch,
+  } = useQuery(GET_CRASH, {
     variables: { crashId },
   });
   const {
```

Comment:
you can use `years_of_life_lost` from the `crashData` and ditch the code that calculates this metric from people. looks like there will be a nice chunk of code to slash, actually ✂️

Reply:
ahh thanks for catching this! i meant to circle back to it and then completely forgot :face_with_spiral_eyes: